README.md

```diff
@@ -1 +1,120 @@
-# book-appointment_back-end
\ No newline at end of file
+# BOOK_APPOINTMENT-BACK-END
+
+This is backend API for the **BOOK_APPOINTMENT-Front-End** UI, built with Ruby-on-Rails and Postgresql. This API manages reservations for car test runs across various locations.
+
+## Built with
+
+- Ruby version 3.1.2
+- Rails (RoR) version 7.0.4
+- Postgresql
+- Rubocop
+
+## Front-End
+
+You can access the front-end of this application via [this link](https://github.com/jahongiry/book-appointment_front-end).
+
+## SetUp
+
+To get a local copy up and running, follow these steps:
+
+- You should have **_Ruby_** installed on your computer.
+- You should also have **_Rails_** installed on your computer.
+- Clone this repository into your computer by running the following command in your terminal:
+
+```bash
+> git clone git@github.com:jahongiry/book-appointment_back-end.git
+> cd book-appointment-backend
+```
+
+- Run this command in the cloned directory, to install required gems:
+
+```
+> bundle install
+```
+
+- Then create the posgreSQL database needed for this app, run the migrations and seed data with the following commands:
+
+```bash
+> rails db:create
+> rails db:migrate
+> rails db:seed
+```
+
+- Run `Rubocop -A` and fix linters errors, if any.
+
+* Run the project in your browser:
+
+```
+rails server
+```
+
+## Run Tests
+
+- To run tests, Run the following command on your terminal
+
+```
+rspec spec
+```
+
+## Deployment
+
+The server was deployed on heroku, and runs on [this domain](https://stark-badlands-38572.herokuapp.com/api/v1/hotels).
+The above endpoint permits you to see all cars available for reservation. Checkout the api documentation below for the other endpoints.
+
+## API Documentation
+
+Visit [this endpoint]() to see a complete documentation of all available endpoints for this API.
+We hope this helps you find your way around our API.
+
+## Authors
+
+👤 **Assad Isah**
+
+- GitHub: [@nottherealalanturing](https://github.com/nottherealalanturing)
+- LinkedIn: [Assad Isah](https://linkedin.com/in/assadisah)
+
+<hr>
+
+👤 **Emirjeta Veisllari**
+
+- GitHub: [@myelin0](https://github.com/myelin0)
+- LinkedIn: [emirjeta-veisllari](https://www.linkedin.com/in/emirjeta-veisllari/)
+- Twitter: [emirjeta-veisllari](https://www.linkedin.com/in/emirjeta-veisllari/)
```

Comment:
- GitHub: [@myelin0](https://github.com/myelin0)
- Twitter: [@Amy_Albania](https://twitter.com/Amy_albania)
- LinkedIn: [@Emirjeta Veisllari](https://www.linkedin.com/in/emirjeta-veisllari/)